spec/amp-actions-and-events.md

```diff
@@ -143,3 +148,20 @@ For example, the following is possible in AMP.
     <td>Hides the referenced user notification element</td>
   </tr>
 </table>
+
+
+## `AMP` target
```

Comment:
That's the agreement we came up with between this action and binder. It's kind of a unique target: not supported by any element. Capital case also seemed to have a lower chance of collision with existing IDs.